services/web-server/src/servers/createApp.js

```diff
@@ -19,7 +19,7 @@ module.exports = async ({ cfg, strategies, AuthorizationCode, AccessToken, auth,
 
   app.set('trust proxy', cfg.server.trustProxy);
   app.set('view engine', 'ejs');
-  app.set('views', 'src/views');
+  app.set('views', __dirname + '../views');
```

Comment:
Does `__dirname` end in a slash? To avoid errors like "path/to/directory../views", should we use `path.join`?

Reply:
Good catch.  Probably `path.resolve`, really.